NSS-Day-Cohort-53/giffygram-spazzy-kermits
# File Structure updated

#### Changes Made
1. Added all new file names to directory.

​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout sh-directoryStructure
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Check all file names for correct spelling, formatting and placement
    > Example: When you pull down the file the structure should be what is agreed on in the chart
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
